data/Mohd-Wamid.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "Mohd Wamid",
+    "bio": "Aspiring Full Stack Developer.",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/Mohd-Wamid",
+            "icon": "FaGithub"
+        },
+        {
+            "name": "Follow me on Instagram",
+            "url": "https://instagram.com/mohdwamid",
+            "icon": "FaInstagram"
+        },
+        {
+            "name": "Follow me on LinkedIn",
+            "url": "https://www.linkedin.com/in/mohd-wamid-ba6a2a251",
+            "icon": "FaLinkedin"
+        },
+        {
+            "name": "Follow me on Facebook",
+            "url": "https://www.facebook.com/profile.php?id=100004495575740",
+            "icon": "FaFacebook"
+        },
+        {
+            "name":" My leetcode Account",
```

Minor suggestion to consider removing white space.
```suggestion
            "name":"My leetcode Account",
```